src/Kunstmaan/MediaBundle/Helper/ManipulateImageService.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Kunstmaan\MediaBundle\Helper;
+
+use Kunstmaan\MediaBundle\Entity\CroppableMediaLink;
+use Kunstmaan\MediaBundle\Entity\Media;
+use Kunstmaan\UtilitiesBundle\Helper\Slugifier;
+use Liip\ImagineBundle\Service\FilterService;
+
+class ManipulateImageService
+{
+    /** @var Slugifier */
+    private $slugifier;
+
+    /** @var FilterService */
+    private $filterService;
+
+    public function __construct(Slugifier $slugifier, FilterService $filterService)
+    {
+        $this->slugifier = $slugifier;
+        $this->filterService = $filterService;
+    }
+
+    public function manipulateOnTheFly(CroppableMediaLink $croppableMediaLink, string $view = '', string $filter = 'optim'): string
```

Comment:
Maybe call this class `ImageCropService`? Or keep the current classname but rename this method to `crop`/`cropImage`?